openapi/components/schemas/QuoteReactivateOrder.yaml

```
@@ -0,0 +1,493 @@
+type: object
+title: QuoteReactivateOrder
+required:
+  - websiteId
+  - customerId
+  - items
+  - action
+  - orderId
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  type:
+    description: Specifies the type of the quote. A quote can be a subscription or a one-time purchase.
+    type: string
+    readOnly: true
+    enum:
+      - subscription-order
+      - one-time-order
+  action:
+    description: |-
+      Action of the quote for churned/canceled order reactivation.
+      On accept quote new invoice is issued to be and linked to `invoiceId`. Quote items are applied to the order.
+      The related order will have `pending` status.
+      Once the invoice is paid, the order becomes active on `settings.reactivate.effectiveTime` or immediately if `settings.reactivate.effectiveTime` is omitted.
+    type: string
+    enum:
+      - reactivate
+    example: reactivate
+  acceptanceConditions:
+    type: array
+    writeOnly: true
+    description: If conditions are fulfilled the quote is automatically accepted.
```

suggestion: for readability 
```suggestion
    description: Conditions that must be met for the quote to be accepted.
```